language/vm/src/deserializer.rs

```diff
@@ -1097,16 +1094,44 @@ fn load_function_def(cursor: &mut VersionedCursor) -> BinaryLoaderResult<Functio
     let flags = cursor.read_u8().map_err(|_| {
```

Comment:
Something that isn't done in the old version either, but we should check for bits that are set that make no sense. I think it's definitely a bug in the current version, since if add meaning to the bits, suddenly a bunch of modules on chain might break.

Reply:
Will add the check

Reply:
Check added in f4a3f50.